core/controller/src/main/scala/whisk/core/entitlement/Entitlement.scala

```diff
@@ -328,15 +328,17 @@ protected[core] abstract class EntitlementProvider(config: WhiskConfig, loadBala
   private def checkConcurrentUserThrottle(user: Identity, right: Privilege, resources: Set[Resource])(
     implicit transid: TransactionId): Future[Unit] = {
     if (right == ACTIVATE && resources.exists(_.collection.path == Collection.ACTIONS)) {
-      checkThrottleOverload(concurrentInvokeThrottler.check(user))
+      checkThrottleOverload(concurrentInvokeThrottler.check(user), user)
     } else Future.successful(())
   }
 
-  private def checkThrottleOverload(throttle: Future[RateLimit])(implicit transid: TransactionId): Future[Unit] = {
+  private def checkThrottleOverload(throttle: Future[RateLimit], user: Identity)(
+    implicit transid: TransactionId): Future[Unit] = {
     throttle.flatMap { limit =>
       if (limit.ok) {
         Future.successful(())
       } else {
+        logging.info(this, s"user '${user.namespace}' has exceeded its throttle limit")
```

Comment:
i'd drop "user" prefix, ` +        logging.info(this, s"'${user.namespace}' has exceeded its throttle limit") `

Reply:
here we don't know if it's the concurrent limit, or per minute limit - anyway to easily recover that? perhaps the RateLimit can have a descriptor to emit here?